package.json

```diff
@@ -194,8 +194,9 @@
     "lint-code": "eslint \"{example,features,scripts,src,test}/**/*.js\"",
     "lint-autofix": "eslint --fix \"{example,features,scripts,src,test}/**/*.js\"",
     "lint": "yarn run lint-code && yarn run lint-dependencies",
-    "prefeature-test": "yarn run build-local",
-    "prepublish": "yarn run build-local",
+    "postinstall": "node scripts/build-local.js",
```

Comment:
Thoughts on calling the script something like `node scripts/build-non-release.js`?